server/controllers/configurationsController/configurationsController.test.js

```diff
@@ -0,0 +1,175 @@
+import ConfigurationsController from '.'
+import { createRequestWithUser, createResponse } from '../../../test/fixtures'
+
+describe('ConfigurationsController', () => {
+  describe(ConfigurationsController.create, () => {
+    const newConfiguration = { body: { name: 'new matrix', owner: 'owl' } }
+    const request = createRequestWithUser(newConfiguration)
+    const response = createResponse()
+
+    describe('When successful', () => {
+      it('sends a status of 200 with a data property', async () => {
+        request.app.locals.appDb.insertOne.mockResolvedValueOnce({
+          insertedId: 'eagle',
+        })
+
+        await ConfigurationsController.create(request, response)
+
+        expect(response.status).toHaveBeenCalledWith(200)
+        expect(response.json).toHaveBeenCalledWith({
+          data: { insertedId: 'eagle' },
+        })
+      })
+    })
+
+    describe('when unsuccessful', () => {
+      it('returns a 400 error with an error message', async () => {
+        request.app.locals.appDb.insertOne.mockImplementation(() => {
+          new Error('some error')
+        })
+
+        await ConfigurationsController.create(request, response)
+
+        expect(response.status).toHaveBeenCalledWith(400)
+        expect(response.json).toHaveBeenCalledWith({
+          error: new Error('some error'),
+        })
+      })
+    })
+  })
+
+  describe(ConfigurationsController.edit, () => {
```

Comment:
I think this should be `update` instead of `edit` based on the other controllers.

Reply:
I am confused, this is inconsistent with what is in the ChartsController and that's that pattern we are implementing